src/smc-webapp/project_files.cjsx

```diff
@@ -1520,7 +1524,7 @@ exports.ProjectFiles = rclass ({name}) ->
         shift_is_down : false
 
     componentDidMount: ->
-        @props.redux.getActions('billing')?.update_customer()
+        window.setTimeout(@props.redux.getActions('billing')?.update_customer, 200)
```

Comment:
COMMENTS About this line:  I always question setTimeout when I see it, and always feel happy when I can eliminate it.  Please at least explain why this is needed.   

Also, I usually prefer using `await delay(200)` if possible, where `{delay} = require('awaiting')`.  Generally, the code is more readable, and it also avoids changing `this` (whereas setTimeout does).

I also don't explicitly use `window.setTimeout` anywhere -- why put `window.` explicitly?   E.g., node also has setTimeout but no window.  By putting `window.setTimeout` in code you're needlessly making it so it can't run in node without defining window globally, right?

Reply:
Oh, no reason to use window. Changed to just `setTimeout`.

I didn't want to look too deeply in this case, but it's just to defer cascading changes that resulted from this action. It should probably be moved elsewhere but at least this way it doesn't affect file listing responsiveness as much.

Before it would do this after mount but before drawing anything
<img width="631" alt="Screen Shot 2019-03-21 at 2 04 25 PM" src="https://user-images.githubusercontent.com/618575/54840899-3ed7b580-4c8b-11e9-9a9e-aefe5e5e58f5.png">

Now it draws and does the update later.
<img width="300" alt="Screen Shot 2019-03-21 at 2 09 05 PM" src="https://user-images.githubusercontent.com/618575/54840898-3ed7b580-4c8b-11e9-93df-640b46f75ceb.png">

Reply:
That's a good explanation.  Can you add a comment saying basically that?